README.md

```diff
@@ -1,2 +1,30 @@
-# scraped_page
+# ScrapedPage
+
 Write declarative scrapers in Ruby
+
+## Usage
+
+Create a subclass of `ScrapedPage` for each page you wish to scrape.
+
+```ruby
+require 'scraped_page'
+
+class ExamplePage < ScrapedPage
+  field :title do
+    noko.at_css('h1').text
```

Comment:
We probably want some explanation somewhere regarding what `noko` is here and where it comes from (not necessarily right now, but noting that this will be a stumbling point.) I guess it's a little harder to insert a TODO: into a README than into code…